src/applications/vaos/services/appointment/transformers.js

```diff
@@ -258,6 +258,35 @@ function getRequestedPeriods(appt) {
   return requestedPeriods.sort((a, b) => (a.start < b.start ? -1 : 1));
 }
 
+function setVideoData(appt) {
+  if (
+    getAppointmentType(appt) !== APPOINTMENT_TYPES.vaAppointment ||
+    !isVideoVisit(appt)
+  ) {
+    return null;
+  }
+
+  const videoData = appt.vvsAppointments[0];
+  return {
+    facilityId: appt.sta6aid || appt.facilityId,
+    providers: videoData.providers
+      ?.filter(provider => !!provider.name)
+      .map(provider => ({
+        name: provider.name,
+        display: `${provider.name.firstName} ${provider.name.lastName}`,
+      })),
+    kind: videoData.appointmentKind,
+    url: getVideoVisitLink(appt),
+    isAtlas: !!videoData.tasInfo,
+    atlasLocation: videoData.tasInfo
+      ? transformATLASLocation(videoData.tasInfo)
```

Comment:
With this kind of refactoring I do want to leave basic locations/addresses in FHIR format wherever possible.